entrypoint.sh

```diff
@@ -31,12 +31,18 @@ format_diff() {
 
 CHECK_PATH="$1"
 FALLBACK_STYLE="$2"
+EXCLUDE_REGEX="$3"
 
-# Install clang-format
-echo "Installing clang-format-$CLANG_FORMAT_VERSION"
-apt-get update && apt-get install -y --no-install-recommends clang-format-"$CLANG_FORMAT_VERSION"
+# Set the regex to an empty string regex if nothing was provided
+if [ -z "$EXCLUDE_REGEX" ]; then
+	EXCLUDE_REGEX="^$"
+fi
+
+# # Install clang-format
+# echo "Installing clang-format-$CLANG_FORMAT_VERSION"
+# apt-get update && apt-get install -y --no-install-recommends clang-format-"$CLANG_FORMAT_VERSION"
```

Comment:
```suggestion
# Install clang-format
echo "Installing clang-format-$CLANG_FORMAT_VERSION"
apt-get update && apt-get install -y --no-install-recommends clang-format-"$CLANG_FORMAT_VERSION"
```

Reply:
Whoops!  Sorry, I didn't mean to commit those.  I'd commented them out to test it locally. 😬

Reply:
All good 🙂